src/Utils.ts

```diff
@@ -23,4 +23,11 @@ export namespace Utils {
       .get('pythonPath');
     return pypiPath ? `${pypiPath}` : 'python';
   }
+
+  export function getGoExecutable(): string {
+    const goPath: string = vscode.workspace
+      .getConfiguration('go')
+      .get('goPath');
+    return goPath ? `${goPath}` : 'go';
+  }
```

Comment:
Isn't vscode settings `go.goPath` referes to [GOPATH](https://golang.org/doc/gopath_code.html#GOPATH) which holds all go modules & build artifacts?